web-server/v0.4/utils/datastore.js

```diff
@@ -1,5 +1,5 @@
 export default { 
-    index_prun_indices: "run.*",
+    run_indices: ".run.*",
     prefix: "dsa.pbench",
     elasticsearch: "//elasticsearch.example.com:9280/",
     production: "//pbench.example.com"
```

Comment:
Can we rename "`production`" to "`pbench_server`"?  That might make more sense since we run this code in production, staging, and other test environments.